tests/statespace_testfixture.h

```diff
@@ -793,6 +793,61 @@ void TestMeasurementLarge() {
   }
 }
 
+template <typename StateSpace>
+void TestInvalidStateSize() {
+  using fp_type = typename StateSpace::fp_type;
+  using State = typename StateSpace::State;
+
+  std::mt19937 rgen(1);
+
+  unsigned num_qubits = 5;
+
+  StateSpace state_space(num_qubits, 1);
+  uint64_t raw_size =
+      state_space.MinimumRawSize(2 * (uint64_t{1} << num_qubits)) - 1;
+  std::vector<fp_type> vec(raw_size, 0);
+  State state = state_space.CreateState(vec.data(), vec.size());
+
+  EXPECT_FALSE(state_space.SetAllZeros(state));
+  EXPECT_FALSE(state_space.SetStateUniform(state));
+  EXPECT_FALSE(state_space.SetStateZero(state));
+  EXPECT_FALSE(state_space.Multiply(1.0, state));
+  EXPECT_FALSE(!std::isnan(state_space.Norm(state)));
+  EXPECT_FALSE(!state_space.PartialNorms(state).empty());
```

Comment:
`EXPECT_FALSE(!<expression>)` can be replaced with `EXPECT_TRUE(<expression>)`.

Reply:
I just wanted to be consistent with other lines expecting false in the case of an error.